src/test/java/kirby/HandleTimeTest.java

```diff
@@ -0,0 +1,31 @@
+package kirby;
+
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+import static kirby.HandleTime.fromStringToDate;
+import static kirby.HandleTime.isValidDate;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class HandleTimeTest {
+    @Test
+    public void negativeCheckDate() {
```

Comment:
Perhaps you could use a different word instead of "negative" to describe this test? I found it a bit confusing to understand the function from the name alone.

Reply:
Since you are testing `isValidDate()` perhaps naming it like this would be more clear?
```suggestion
    public void isValidDate_false() {
```

Reply:
Thanks :)